web/tendrl/user_password_edit_dialog.rst

```diff
@@ -0,0 +1,67 @@
+Change user password at user "Edit" page
+*****************************************
+
+:authors: 
+          - mkudlej@redhat.com
+
+Description
+===========
+
+Change user password at user "Edit" page for these user types:
+ 1. *Admin* user
+ 2. *Power User* user
+ 3. *Read-only* user
+
+Design describing "Edit" page `https://redhat.invisionapp.com/share/KNB25OEQT#/screens/226063805`
+
+Setup
+=====
+
+#. Login user into Tendrl, see :doc:`/web/tendrl/login_positive`
+#. Create 3 users mentioned in Description, see :doc:`/web/tendrl/user_create`
+
+Test Steps
+==========
+
+.. test_action::
+   :step:
+       Click on *Admin* -> *Users* in left menu.
+   :result:
+       Page with user list is open.
+
+.. test_action::
+   :step:
+       Click on *Edit* button for user.
+   :result:
+       "Edit" page should be open.
+
+.. test_action::
+   :step:
+       Input password to *Change Password* and same string to *Confirm Password*.
+   :result:
+       Password in both inputs are not visible and instead of them are dots.
+
+.. test_action::
+   :step:
+       Click on icon next to *Change Password* and *Confirm Password* input.
```

Comment:
There is no confirm password input in the design doc you linked.

Reply:
You're right. I've filed bug https://github.com/Tendrl/ui/issues/586 but I expect that it *Confirm Password* will stay there. I prefer to leave it here now and change according Ju's comment.